README.md

```diff
@@ -184,7 +184,7 @@ Installed packages, channel pecl.php.net:
 Package version state
 apcu    5.1.19  stable
 redis   5.3.2   stable
-xdebug  2.9.8   stable
```

Comment:
Is that only the difference? Can you run the script?

Reply:
https://travis-ci.com/github/spryker/docker-php/jobs/490024642#L3747 - here are the latest versions. And redis like example is different.